haskell/ghc_bindist.bzl

```diff
@@ -373,10 +373,14 @@ haskell_toolchain(
 
     if os == "windows":
         # These libraries cause linking errors on Windows when linking
-        # pthreads, due to libwinpthread-1.dll not being loaded.
-        execute_or_fail_loudly(ctx, ["rm", "mingw/lib/gcc/x86_64-w64-mingw32/7.2.0/libstdc++.dll.a"])
-        execute_or_fail_loudly(ctx, ["rm", "mingw/x86_64-w64-mingw32/lib/libpthread.dll.a"])
-        execute_or_fail_loudly(ctx, ["rm", "mingw/x86_64-w64-mingw32/lib/libwinpthread.dll.a"])
+        # pthreads, due to libwinpthread-1.dll not being loaded. It's
+        # hard to guesss the paths of these libraries, and we can't
+        # assume `find` is available, so we delete the files on
+        # a best-effort basis (no error if we can't find them).
```

Comment:
We could use `dir` instead of `find` similar to how it's done [here](https://github.com/tweag/rules_haskell/blob/3987c494f7a3b72805855246a844a28880dd0b15/haskell/cabal.bzl#L1267). E.g. `dir /sb mingw/libwinpthread.dll.a`.